auslib/db.py

```diff
@@ -1649,7 +1670,7 @@ def getRawMatches():
                 self.log.debug("%s doesn't match %s", rule['osVersion'], updateQuery['osVersion'])
```

Comment:
Thanks - I did that and updated 3 of the osVersion tests. I was a bit on the fence about this, since this behavior is already tested using systemCapabilities, _but_ I thought I'd it to specifically test the behavior of osVersion matching.